examples/ffmpeg_vad_streaming/index.js

```diff
@@ -59,59 +60,74 @@ const model_load_end = process.hrtime(model_load_start);
 console.error('Loaded model in %ds.', totalTime(model_load_end));
 
 if (args['lm'] && args['trie']) {
-	console.error('Loading language model from files %s %s', args['lm'], args['trie']);
-	const lm_load_start = process.hrtime();
-	model.enableDecoderWithLM(args['alphabet'], args['lm'], args['trie'],
-		LM_ALPHA, LM_BETA);
-	const lm_load_end = process.hrtime(lm_load_start);
-	console.error('Loaded language model in %ds.', totalTime(lm_load_end));
+  console.error('Loading language model from files %s %s', args['lm'], args['trie']);
+  const lm_load_start = process.hrtime();
+  model.enableDecoderWithLM(args['alphabet'], args['lm'], args['trie'],
+    LM_ALPHA, LM_BETA);
+  const lm_load_end = process.hrtime(lm_load_start);
+  console.error('Loaded language model in %ds.', totalTime(lm_load_end));
 }
 
-const vad = new VAD(VAD.Mode.NORMAL);
-const voice = {START: true, STOP: false};
-let sctx = model.setupStream(150, 16000);
-let state = voice.STOP;
+// Default initial allocation = 3 seconds := 150
+const PRE_ALLOC_FRAMES = 150;
+
+// Default is 16kHz
+const AUDIO_SAMPLE_RATE = 16000;
+
+const VAD_MODE = VAD.Mode.NORMAL;
+// const VAD_MODE = VAD.Mode.LOW_BITRATE;
+// const VAD_MODE = VAD.Mode.AGGRESSIVE;
+// const VAD_MODE = VAD.Mode.VERY_AGGRESSIVE;
+
+// Debounce time
+const DEBOUNCE_TIME = 20;
```

Comment:
Maybe worth commenting the use of that and the unit to help understand ; I guess it's in `ms` ?